README.md

```diff
@@ -48,16 +51,17 @@ cp ninja <a_directory_in_your_path>
 ## Buildings
 
 This is the simple one.
+You need pkg-config to be installed
```

Comment:
It would be better to make kiwix-build install the pkg-config package if needed.
Adding the package name in the `COMMON` entries of the `PACKAGE_NAME_MAPPERS` dictionaries in `kiwix-build.py` will be enough.